src/editor.c

```diff
@@ -1906,9 +2002,27 @@ static gchar *find_calltip(const gchar *word, GeanyFiletype *ft)
 
 	g_return_val_if_fail(ft && word && *word, NULL);
 
-	/* use all types in case language uses wrong tag type e.g. python "members" instead of "methods" */
-	tags = tm_workspace_find(word, NULL, tm_tag_max_t, NULL, ft->lang);
-	if (tags->len == 0)
+	tags = get_scoped_tags(editor, pos - strlen(word));
+	if (tags)
+	{
+		GPtrArray *filtered = g_ptr_array_new();
+
+		foreach_ptr_array(tag, i, tags)
+		{
+			if (strcmp(tag->name, word) == 0)
+				g_ptr_array_add(filtered, tag);
```

Comment:
No it does not, it searches members of a scope, doesn't filter by names (it supports getting members without specifying a name).
And this part is basically just copied from L880 onwards